luxjs/grid/grid.js

```diff
@@ -171,11 +190,12 @@
                         });
 
                         defer.promise.then(function() {
+                            var alertMsg = scope.selected.length + ' ' + stateName + '.';
                             if (success) {
                                 getPage(scope, api);
-                                $lux.messages.success('Successfully deleted ' + stateName + ' ' + results);
+                                $lux.messages.success('Successfully deleted ' + alertMsg);
                             } else
-                                $lux.messages.error('Error while deleting ' + stateName + ' ' + results);
+                                $lux.messages.error('Error while deleting ' + alertMsg);
 
                             modal.hide();
                         });
```

Comment:
if an error occurs (status code 400 or higher) no message is displayed. The reason is you are not handling the error, you need to pass a second function to ``then`` for handling this.